### PR TITLE
Adding support for fzf-lua in the quickswitch command

### PR DESCRIPTION
**Pull request by saccarosium**
As mentioned in #68

### CODE REVIEW

File: lua/obsidian/command.lua [hunk 1]
```diff
@@ -1,5 +1,3 @@
-local Path = require "plenary.path"
-local Job = require "plenary.job"
```
Comment (epwalsh) [inline lua/obsidian/command.lua:2]: I'm guessing you didn't mean to remove these?

**Review by epwalsh — CHANGES_REQUESTED** [review]
LGTM, just one issue

File: lua/obsidian/command.lua (hunk 1 above)
Comment (saccarosium) [reply lua/obsidian/command.lua:2]: Strange I don't really know what happened

**Comment by saccarosium** [comment]
Should be good

**Review by epwalsh — APPROVED** [review]
Thank you!!

**Pull request merged**